README.md

```diff
@@ -6,7 +6,7 @@ gocraft/dbr provides additions to Go's database/sql for super fast performance a
 
 ```go
 // create a connection
-conn, _ := dbr.Open("postgres", "...")
+conn, _ := dbr.Open("postgress", "your_user:password@tcp(127.0.0.1:5432)/your_db", nil)
```

Comment:
IMO: ignoring errors is not good example :)

Reply:
I think that's my fault, but yes, we should demo error handling here IMO.